home/gettingstarted/0.7/executors.md

```diff
@@ -155,9 +155,34 @@ And the final output:
         (ci = <20000.02 kB, 20000.02 kB>, significance = 1.0E-10)
 
 
-You can take a look at the ScalaMeter API or source code to figure out how different measurers work in more detail.
+### Invocation Count
+
```

Comment:
Please add an introductory sentence on what this measurer is, and what it is used for. Something like:

    The invocation count measurers are used to count the total number of invocations
    of a specific method, or a group of methods.